daprdocs/content/en/getting-started/quickstarts/jobs-quickstart.md

```diff
@@ -0,0 +1,542 @@
+== APP - job-service == Registered job handler for:  R2-D2
+== APP - job-service == Registered job handler for:  C-3PO
+== APP - job-service == Registered job handler for:  BB-8
+== APP - job-service == Starting server on port: 6200
+== APP - job-service == Job scheduled:  R2-D2
+== APP - job-service == Job scheduled:  C-3PO
+== APP - job-service == 2024/07/17 18:09:59 job:{name:"C-3PO"  due_time:"10s"  data:{value:"{\"droid\":\"C-3PO\",\"Task\":\"Memory Wipe\"}"}}
+== APP - job-scheduler == Get job response:  {"droid":"C-3PO","Task":"Memory Wipe"}
+== APP - job-service == Job scheduled:  BB-8
+== APP - job-service == 2024/07/17 18:09:59 job:{name:"BB-8"  due_time:"15s"  data:{value:"{\"droid\":\"BB-8\",\"Task\":\"Internal Gyroscope Check\"}"}}
+== APP - job-scheduler == Get job response:  {"droid":"BB-8","Task":"Internal Gyroscope Check"}
+== APP - job-scheduler == Deleted job:  BB-8
+```
+
+After 5 seconds, the terminal output should present the `R2-D2` job being processed:
+
+```text
+== APP - job-service == Starting droid: R2-D2
+== APP - job-service == Executing maintenance job: Oil Change
+```
+
+After 10 seconds, the terminal output should present the `C3-PO` job being processed:
+
+```text
+== APP - job-service == Starting droid: C-3PO
+== APP - job-service == Executing maintenance job: Memory Wipe
+```
+
+Once the process has completed, you can stop and clean up application processes with a single command.
+
+```bash
+dapr stop -f .
+```
+
+### What happened?
+
+When you ran `dapr init` during Dapr install:
+
+- The `dapr_scheduler` control plane was started alongside other Dapr services.
+- [The `dapr.yaml` Multi-App Run template file]({{< ref "#dapryaml-multi-app-run-template-file" >}}) was generated in the `.dapr/components` directory.
+
+Running `dapr run -f .` in this Quickstart started both the `job-scheduler` and the `job-service`. In the terminal output, you can see the following jobs being scheduled, retrieved, and deleted.
+
+- The `R2-D2` job is being scheduled.
+- The `C-3PO` job is being scheduled.
+- The `C-3PO` job is being retrieved.
+- The `BB-8` job is being scheduled.
+- The `BB-8` job is being retrieved.
+- The `BB-8` job is being deleted.
+- The `R2-D2` job is being executed after 5 seconds.
+- The `R2-D2` job is being executed after 10 seconds.
+
+#### `dapr.yaml` Multi-App Run template file
+
+Running the [Multi-App Run template file]({{< ref multi-app-dapr-run >}}) with `dapr run -f .` starts all applications in your project. In this Quickstart, the `dapr.yaml` file contains the following:
+
+```yml
+version: 1
+apps:
+  - appDirPath: ./job-service/
+    appID: job-service
+    appPort: 6200
+    daprGRPCPort: 6281
+    appProtocol: grpc
+    command: ["go", "run", "."]
+  - appDirPath: ./job-scheduler/
+    appID: job-scheduler
+    appPort: 6300
+    command: ["go", "run", "."]
+```
+
+#### `job-service` app
+
+The `job-service` application creates handlers for the job service. 
```

Comment:
Replace with: The `job-service` application creates service invocation handlers to manage the lifecycle of the job (schedule, get, delete).